services/storage/src/simcore_service_storage/simcore_s3_dsm.py

```diff
@@ -276,15 +276,19 @@ async def create_file_upload_links(
                 conn, parse_obj_as(SimcoreS3FileID, file_id)
             )
 
-        # ensure file is deleted first in case it already exists
-        await self.delete_file(
-            user_id=user_id,
-            file_id=file_id,
-            # NOTE: bypassing check since the project access rights don't play well
-            # with collaborators
-            # SEE https://github.com/ITISFoundation/osparc-simcore/issues/5159
-            enforce_access_rights=False,
-        )
+        if (
+            not is_directory
+        ):  # NOTE: Delete is not needed for directories that are synced via an external tool (rclone/aws s3 cli).
+            # ensure file is deleted first in case it already exists
+            # https://github.com/ITISFoundation/osparc-simcore/pull/5108
+            await self.delete_file(
+                user_id=user_id,
+                file_id=file_id,
+                # NOTE: bypassing check since the project access rights don't play well
+                # with collaborators
+                # SEE https://github.com/ITISFoundation/osparc-simcore/issues/5159
+                enforce_access_rights=False,
+            )
```

Comment:
We should open an issue to replace in storage the copy of projects with the AWS CLI sync if it proves reliable.